src/TLuaInterpreter.cpp

```diff
@@ -11390,25 +11381,14 @@ int TLuaInterpreter::Echo(lua_State* L)
     }
     QString displayText{lua_tostring(L, n)};
 
-    auto console = CONSOLE(L, consoleName);
-    if (console == host.mpConsole) {
-        host.mpConsole->buffer.mEchoingText = true;
```

Comment:
Is the `TBuffer::mEchoingText` flag being set for the MAIN `TConsole` only in the replacement code - as it isn't being handled here?